srcpkgs/Logseq/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'Logseq'
+pkgname=Logseq
+version=0.9.8
+revision=1
+archs="x86_64"
+short_desc="Open-source platform for knowledge sharing and management"
+maintainer="Viachaslau Ravinski <shnapsx@gmail.com>"
+license="AGPL-3.0-only"
+homepage="https://github.com/logseq/logseq"
+distfiles="https://github.com/logseq/logseq/releases/download/${version}/logseq-linux-x64-${version}.zip"
```

Comment:
this will not be accepted as is, using the binary release when the source is available

Reply:
I haven't considered compiling from sources yet. I'll think about the PR in this case.